docs/ref/models/options.txt

```diff
@@ -397,3 +403,20 @@ Django quotes column and table names behind the scenes.
         verbose_name_plural = "stories"
 
     If this isn't given, Django will use :attr:`~Options.verbose_name` + ``"s"``.
+
+Read-only ``Meta`` attributes
+=============================
+
+``label``
+---------
+
+.. attribute:: Options.label
+
+    Representation of the object, returns ``app_label.object_name``.
```

Comment:
maybe we could make `app_label.object_name` links to the respective attributes and also give an example like 'polls.Question'